File: src/main/kotlin/top/charjin/shoppingserver/entity/OsCart.java
package top.charjin.shoppingserver.entity;

import java.io.Serializable;

public class OsCart implements Serializable {
    private static final long serialVersionUID = 1L;
    /**
     * 主外键,用户id
     */
    private Integer userId;
    /**
     * 主外键,商品id
     */
    private Integer goodsId;
    /**
     * 该商品的数量
     */
    private Integer number;
    private String plan;

    public Integer getUserId() {
        return userId;
    }

    public void setUserId(Integer userId) {
        this.userId = userId;
    }

    public Integer getGoodsId() {
        return goodsId;
    }

    public void setGoodsId(Integer goodsId) {
        this.goodsId = goodsId;
    }

    public Integer getNumber() {
        return number;
    }

    public void setNumber(Integer number) {
        this.number = number;
    }

    public String getPlan() {
        return plan;
    }

    public void setPlan(String plan) {
        this.plan = plan;
    }

    @Override
    public String toString() {
        StringBuilder sb = new StringBuilder();
        sb.append(getClass().getSimpleName());
        sb.append(" [");
        sb.append("Hash = ").append(hashCode());
        sb.append(", userId=").append(userId);
        sb.append(", goodsId=").append(goodsId);
        sb.append(", number=").append(number);
        sb.append(", plan=").append(plan);
        sb.append("]");
        return sb.toString();
    }
}

File: src/main/kotlin/top/charjin/shoppingserver/entity/OsCart.kt
package top.charjin.shoppingserver.entity

import java.io.Serializable

class OsCart : Serializable {
    var id: Int? = null

    /**
     * 外键,用户id
     */
    var userId: Int? = null

    /**
     * 外键,商品id
     */
    var goodsId: Int? = null

    var number: Int? = null

    override fun toString(): String {
        val sb = StringBuilder()
        sb.append(javaClass.simpleName)
        sb.append(" [")
        sb.append("Hash = ").append(hashCode())
        sb.append(", id=").append(id)
        sb.append(", userId=").append(userId)
        sb.append(", goodsId=").append(goodsId)
        sb.append(", number=").append(number)
        sb.append("]")
        return sb.toString()
    }

    companion object {

        private const val serialVersionUID = 1L
    }
}